test/controller/objecting.entity.name.controller.spec.unit.ts

```diff
@@ -16,6 +21,9 @@ import {
   OBJECTIONS_OBJECTING_ENTITY_NAME
 } from "../../src/model/page.urls";
 import { COOKIE_NAME } from "../../src/utils/properties";
+import { ObjectionCreate } from "../../src/modules/sdk/objections";
+
+const mockGetObjectCreate = retrieveObjectionCreateFromObjectionSession as jest.Mock
```

Comment:
needs a unit test to check that the error page is shown if the session is not present

Reply:
Added two unit tests one for absent object created and another for the absent session.